src/main.cpp

```diff
@@ -15,16 +15,27 @@ int main(int argc, char** argv) {
         RIP("XDG_RUNTIME_DIR not set!");
 
     // parse some args
-    for (int i = 1; i < argc; ++i) {
-        if (!strcmp(argv[i], "--i-am-really-stupid"))
-            ignoreSudo = true;
-    }
+    if (!strcmp(argv[1], "i-am-really-stupid"))
+        ignoreSudo = true;
+    else if (!strcmp(argv[1], "help"))
+        std::cout << "Available commands:\n";
+        std::cout << "  help                 - show this help text\n";
+        std::cout << "  version              - show version\n";
+        std::cout << "  --i-am-really-stupid - ignore sudo run check\n";
```

Comment:
There seem to be inconsistencies in the code. In one place it's `--i-am-really-stupid`, in another it's `i-am-really-stupid`. I think we should leave it with `--` to maintain backwards compatibility. Also this isn't a usual command so it's fine.